docs/v.list

```diff
@@ -4,29 +4,29 @@
 
 <vars>
 
-    <var name="kotlinVersion" value="1.7.21" type="string"/>
+    <var name="kotlinVersion" value="1.8.0" type="string"/>
 
     <var name="kotlinEapVersion" value="1.8.0-RC2" type="string"/>
 
-    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v1.7.21" type="string"/>
+    <var name="kotlinLatestUrl" value="https://github.com/JetBrains/kotlin/releases/tag/v1.8.0" type="string"/>
 
-    <var name="kotlinReleaseDate" value="November 9, 2022" type="string"/>
+    <var name="kotlinReleaseDate" value="December 28, 2022" type="string"/>
 
     <var name="kotlinEapReleaseDate" value="December 20, 2022" type="string"/>
 
     <var name="coroutinesVersion" value="1.6.4" type="string"/>
 
     <var name="coroutinesEapVersion" value="1.6.0-RC3" type="string"/>
 
-    <var name="minGradleVersion" value="6.7.1" type="string"/>
+    <var name="minGradleVersion" value="6.8.3" type="string"/>
 
-    <var name="maxGradleVersion" value="7.1.1" type="string"/>
+    <var name="maxGradleVersion" value="7.3.3" type="string"/>
 
-    <var name="minAndroidGradleVersion" value="3.6.4" type="string"/>
+    <var name="minAndroidGradleVersion" value="4.1.3" type="string"/>
 
-    <var name="maxAndroidGradleVersion" value="7.0.4" type="string"/>
+    <var name="maxAndroidGradleVersion" value="7.2.1" type="string"/>
 
-    <var name="gradleVersion" value="6.7.1" type="string"/>
+    <var name="gradleVersion" value="7.3" type="string"/>
```

Comment:
@koshachy Maybe we need to replace all usages of this variable with maxGradleVersion